README.md

```diff
@@ -1,20 +1,13 @@
-# Active Travel Infrastructure Platform
+# Plan Your Active Travel Schemes
 
-ATIP lets you design, modify and share active travel infrastructure plans.
+This repository contains the Scheme Sketcher and Scheme Browser, two tools for mapping active travel schemes and viewing them alongside contextual layers. The tools are developed by the Alan Turing Institute in collaboration with Active Travel England.
```

Comment:
Maybe we should add a sentence explaining why the repo is called ATIP?

Reply:
Good point, done